src/Pages/Platforms/rootInfo.tsx

```diff
@@ -1,19 +1,14 @@
-import * as React from 'react'
-import { Col, Row } from 'reactstrap'
+/**
+ * Info that is displayed when no platforms are selected.
+ */
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6263578)